### PR TITLE
Edits to PR #2369

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0d2a60528c/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c0a3338a7a/how-to/).

File: content/dedicated-hosting/view-your-network-device-configuration.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+type: article
+created_date: '2017-12-21'
+created_by: Trevor Becker
+last_modified_date: '2017-01-08'
+last_modified_by: Nate Archer
+product: Dedicated Hosting
+product_url: dedicated-hosting
+---
+
+The MyRackspace portal contains a self-service feature that enables you to download a copy of your firewall or load balancer's running configuration. You can use this feature for your documentation and auditing purposes.
```
Comment (stephamon) [inline content/dedicated-hosting/view-your-network-device-configuration.md]: Is "load balancer's" meant to be possessive or plural?

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-d0c2dbcedb/how-to/). :bow:

**Comment by stephamon** [comment]
👍

**Pull request merged**